packages/dev/core/src/Meshes/Compression/dracoEncoder.types.ts

```diff
@@ -0,0 +1,104 @@
+import type { Nullable } from "core/types";
+
+/**
+ * The available Draco attribute names.
+ */
+export enum DracoAttributeName {
+    POSITION = "POSITION",
+    NORMAL = "NORMAL",
+    COLOR = "COLOR",
+    TEX_COORD = "TEX_COORD",
+    GENERIC = "GENERIC",
+}
+
+/**
+ * Draco encoding method (from EncoderMethod enum in `draco_encoder.ts`).
+ */
+export enum DracoEncoderMethod {
+    /**
+     * Lower compression, but preserves the order of vertices.
+     */
+    SEQUENTIAL = "MESH_SEQUENTIAL_ENCODING",
+    /**
+     * Higher compression, but changes the order of vertices.
+     * NOTE: This will not work with morph targets.
+     */
+    EDGEBREAKER = "MESH_EDGEBREAKER_ENCODING",
+}
```

Comment:
I was initially hesitant to use string unions because I thought they didn't work as well with code completion, but I've just discovered that they do! Amazing! Thanks!